package org.eduardo.exception;

public class dataRecoveryException extends RuntimeException{
    public dataRecoveryException(String message, Throwable cause) {
        super(message, cause);
    }
}
